developers.css

```diff
@@ -40,3 +40,31 @@
 	max-width:140px;
 	float:right;
 }
+
+.guideline:before {
+	content: "?";
+	position: absolute;
+	padding: 1px;
+	background-color: white;
+	border-radius: 5px;
+}
+
+.guideline:hover:before {
+	font-family:"Titillium Web", "Fira Sans", Cantarell, Arial;
+	display: inline;
+	position: absolute;
+	font-size: 16px
+	border: 1px solid black;
+	padding-left: 5px;
+	padding-right: 5px;
+	background-color: white;
+	border-radius: 3px
+}
+
+.ref1:hover:before {content: "Make sure functions and variables that don't need to be public are defined local"}
```

Comment:
Looks good :+1: ! But could you please change it so that you don't mix content with CSS? I would suggest to move this section here that contains the content of the hover boxes into the `devguidelines.html` file inside a `<style>` tag. Otherwise I think I'm fine with this change.